docs/environmentvariables.md

```diff
@@ -1,80 +1,54 @@
-
-#### `LNG_SERVE_PORT`
-
-On which port the Lightning-CLI should serve when running `lng serve` (or `lng dev`). Defaults to auto incrementing depending on available ports, starting at `8080`
-
-#### `LNG_SERVE_PROXY`
-
-Proxies all requests which can't be resolved locally to the given url.
+## Types of Environment Variables
 
-#### `LNG_BUILD_SOURCEMAP`
-
-Instructs the Lightning-CLI whether and how to generate sourcemaps. Possible values: `true`, `false` or `inline`. Defaults to `true`
-
-`true` will generate the sourcemaps in a separate file (`appBundle.js.map`). `inline` will append the sourcemaps to the `appBundle.js` itself as a data URI.
-
-#### `LNG_BUILD_FOLDER`
-
-In which folder the built App (using `lng build`) should be generated. Defaults to `build`
-
-#### `LNG_DIST_FOLDER`
-
-In which folder the standalone, distributable App (using `lng dist`) should be generated. Defaults to `dist`
-
-#### `LNG_AUTO_UPDATE`
-
-Whether or not the Lightning CLI should auto update. Possible values: `true` or `false`, defaults to `true`. **Note**: it's recommended to keep auto updates enabled.
+### NODE_ENV
 
-#### `LNG_LIVE_RELOAD`
+The environment variable `NODE_ENV` is used by several external libraries. You can reference to this environment variable in your App code as `process.env.NODE_ENV`.
 
-Instructs your browser to reload the location when a new app bundle is created (using `lng dev`). When the watcher resolves,
-`document.location.reload()` is called in the browser (tab) that serves your app. Possible value: `true` or `false`. Defaults to `false`
+### Behavior Environment Variables
 
-#### `LNG_LIVE_RELOAD_PORT`
+You can use the following environment variables to customize the behavior of the Lightning CLI:
 
-Live reload communication is driven by WebSockets. `LNG_LIVE_RELOAD_PORT` let's you specify the port it's listening on. Defaults to `8888`
+| Name | Default | Description |
+|---|---|---|
+| `LNG_SERVE_OPEN` | true | Indicates whether or not the Lightning CLI opens a browser window when running `lng serve` or `lng dev`. Possible values: `true`, `false`. |
+| `LNG_SERVE_PORT` | auto-incrementing, start at '8080' | Specifies on which port the Lightning CLI must serve when running `lng serve` or `lng dev`. Auto-incrementing and starting port (see Default) depend on available ports. |
+| `LNG_SERVE_PROXY` | (N.A.) | Proxies all requests that cannot be resolved locally to the given URL. |
+| `LNG_BUILD_SOURCEMAP` | true | Instructs the Lightning CLI whether or not and if so, *how* to generate sourcemaps. Possible values: `true`, `false`, `inline`. The value `true` generates the sourcemaps in a separate file (**appBundle.js.map**). The value `inline` appends the sourcemaps to the **appBundle.js** itself as a data URI. |
+| `LNG_BUILD_FOLDER` | build | Specifies the folder in which the built App (using `lng build`) must be generated. |
```

Comment:
"will be generated"?

Reply:
Agreed. Corrected twice (LNG_BUILD_FOLDER and LNG_DIST_FOLDER).